bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/i18n/luxtronikheatpump.properties

```
@@ -81,6 +88,7 @@ channel-type.luxtronikheatpump.heatingControllerLessTime.label = Heating Control
 channel-type.luxtronikheatpump.heatingControllerMoreTime.label = Heating Controller More Time
 channel-type.luxtronikheatpump.heatingMode.label = Heating Mode
 channel-type.luxtronikheatpump.heatingPowerActualValue.label = Heating Power Actual Value
+channel-type.luxtronikheatpump.powerConsumption.label = Power consumption
```

```suggestion
channel-type.luxtronikheatpump.powerConsumption.label = Power Consumption
```

Concider regenerating I18N properties:
https://www.openhab.org/docs/developer/utils/i18n.html#generating-i18n-properties-file